docker/de/Readme.md

```diff
@@ -16,22 +16,16 @@ $ sh bin/build-docker-de
 
 >**Note:** The `--name` flag allows you to name the container and refer to that name in subsequent commands.
 
+First create a new container with the name `chaincore`:
+
 ```sh
-$ docker run -p 1999:1999 --name chaincore chaincore/developer
+$ docker create -p 1999:1999 --name chaincore chaincore/developer
```

Comment:
I left it as I found it without `-t` (though the missing `-i` is an indicator that it was just out-of-date). 

I think without `-t`, you can't close your session with `ctrl-c`. I'll add it here to be consistent.